### PR TITLE
created select options for city and class in progress

### PR DESCRIPTION
**Pull request by NawalC**
### What changes have you made?

- Started work on city and class dropdown

### CODE REVIEW

**Comment by NawalC** [comment]
@petemcfarlane since we will be having more if statements to include other cities would a loop work better?

File: client/src/components/useFormValidation.js [hunk 1]
```diff
@@ -11,8 +10,9 @@ function useFormValidation(validate, intialState)  {
       ...input,
       [event.target.name]: event.target.value,
     };
-    setInput(updateInput)
-    setIsValid(false)
+    setInput(updateInput);
+    setIsValid(false);
+    console.log(updateInput);
```
Comment (petemcfarlane) [inline client/src/components/useFormValidation.js:15]: Think you accidentally left this in

File: client/src/components/CityDropDownOptions.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            <option value="select">Select</option>
+            {cities.London.map((cohort, index) => (
+              <option key={index} value={cohort}>
+                {cohort}
+              </option>
+            ))}
+          </select>
+        </>
+      );
+    }
```
Comment (petemcfarlane) [inline client/src/components/CityDropDownOptions.js]: 🟢 Each of these 3 `if` blocks look very similar, you _could_ extract the middle bit into a teeny tiny component, or you could get clever with a map, like your existing `Object.keys(cities).map(...)`
Or you could leave it as it is :D

**Review by petemcfarlane — APPROVED** [review]
Nice one! What happens when new classes are added, does someone just have to update the `cities` data?

File: client/src/components/CityDropDownOptions.js (hunk 2 above)
Comment (NawalC) [reply client/src/components/CityDropDownOptions.js]: @petemcfarlane we tried the Object.keys(cities).map(...) option last night but couldn't figure it out. any hints?

Comment (petemcfarlane) [reply client/src/components/CityDropDownOptions.js]: Something like
```js
Object.keys(cities).map((city) => (
  <>
    <label for="userClassId">Choose a Class</label>
    <select name="userClassId" onChange={handleChange}>
      <option value="select">Select</option>
      {cities[city].map((cohort, index) => (
        <option key={index} value={cohort}>
          {cohort}
        </option>
      ))}
    </select>
  </>
))
```

**Pull request merged**